src/App.css

```diff
@@ -60,10 +60,47 @@
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
-  width: 500px;
+  width: 275px;
   background-color: whitesmoke;
-  border: 2px solid, black;
-  border-radius: 0.5rem;
+  border-radius: 0.3rem;
+  padding: 2rem;
+}
+
+.error-list {
+  list-style: disc;
+}
+
+.form-head-section {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 1rem;
+}
+
+.form-header {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 1.5rem;
+}
+
+.form-row {
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  margin: 0.5rem;
+}
+
+.form-label {
+  font-family: "Roboto", sans-serif;
```

Comment:
Out of curiosity how did you choose the font? 

Why is it important to add `, sans-serif` after the font name?